lib/screens/activity_edit_sheet.dart

```diff
@@ -70,6 +74,14 @@ class _ActivityEditSheetState extends State<ActivityEditSheet> {
   }
 
   Widget _buildBody(BuildContext context) {
+    // Before we return the body of the main context/activity sheet, it checks the conditions everytime
```

Comment:
return the result from that function based on the condition.

Reply:
Yes let me fix that using the method.